plugins/modules/nxos_interfaces.py

```diff
@@ -31,100 +13,162 @@
 
 __metaclass__ = type
 
-
 DOCUMENTATION = """
 module: nxos_interfaces
 short_description: Interfaces resource module
 description: This module manages the interface attributes of NX-OS interfaces.
 version_added: 1.0.0
-author: Trishna Guha (@trishnaguha)
+author:
+  - Trishna Guha (@trishnaguha)
+  - Vinay Mulugund (@roverflow)
 notes:
-- Tested against NXOS 7.3.(0)D1(1) on VIRL
-- Unsupported for Cisco MDS
+  - Tested against NXOS 10.4(2) on CML
```

Comment:
We might want to revise the nxos support metric then.
Should be  @dafne qustion!